### DISCUSSION TITLE
Separate local and global chat

### DISCUSSION THREAD

**Discussion by DarkFort** [Zrips/CMI#1754] [Enhancement]
It would be more convenient if local and global chats would be separated. For example, to be able to set different colors on the chat.
For example:
![image](https://user-images.githubusercontent.com/25278617/57697173-4ef08d80-765b-11e9-8a3f-16f7bc558d28.png)
Also, I would like the _/ignore_ command to extend not only to the personal chat (/msg), but also to the global chat.

---

**Comment by DarkFort** [comment]
Also, it would be great to add surveillance to a local chat.

**Comment by RedAlphard** [comment]
I'm interested, but I'd add the variable permissions; That is, those who have that permission can read and write in that chat. A party system on your part would be welcome.

P.S. You solve all my problems.

**Comment by DarkFort** [comment]
I would also like the player to be written about the fact that no one heard his message within a certain radius.

**Comment by DarkFort** [comment]
Also, it is necessary to add a delay in entering messages in the local and global chat, so there was no spam.
